### PR TITLE
Overwrite ssl-config-hail-root secret for dev

### PR DESCRIPTION
**Pull request by vladsaveliev**

### CODE REVIEW

File: build.yaml [hunk 1]
```diff
@@ -154,6 +154,10 @@ steps:
    image:
      valueFrom: create_certs_image.image
    script: |
+     until ! kubectl get secret -n {{ default_ns.name }} ssl-config-hail-root 2>/dev/null
```
Comment (lgruen) [inline build.yaml:157]: Could you add a comment explaining why we're doing this?
(The dev certificates expires after a month.)

Comment (vladsaveliev) [reply build.yaml:160]: Done!

File: build.yaml [hunk 2]
```diff
@@ -154,6 +154,13 @@ steps:
    image:
      valueFrom: create_certs_image.image
    script: |
+     # Create dev certificates, or overwrite the existing ones. The usecase for 
+     # overwriting is to update the dev certificates, that expire after 30 days
```
Comment (lgruen) [inline build.yaml:158]:
```suggestion
     # overwriting is to update the dev certificates, as they expire after 30 days
```

**Review by lgruen — APPROVED** [review]
Thanks!

**Pull request merged**